docker-compose.yaml

```diff
@@ -8,3 +8,10 @@ services:
     environment:
       APPINSIGHTS_CLOUDROLE: fcp-fd-comms-local
       APPINSIGHTS_CONNECTIONSTRING: ${APPINSIGHTS_CONNECTIONSTRING}
+      MESSAGE_QUEUE_HOST: ${MESSAGE_QUEUE_HOST}
+      MESSAGE_QUEUE_USER: ${MESSAGE_QUEUE_USER}
+      MESSAGE_QUEUE_PASSWORD: ${MESSAGE_QUEUE_PASSWORD}
+      COMMS_TOPIC_ADDRESS: ${MESSAGE_QUEUE_SUFFIX}-${DEV_SUFFIX}
+      COMMS_SUBSCRIPTION_ADDRESS: ${MESSAGE_QUEUE_SUFFIX}-${DEV_SUFFIX}
```

Comment:
Makes sense since we use the `DEV_SUFFIX` for Service Bus and the Comms repo is the only service of ours that will use Service Bus? Have made the change, variable is now called `MESSAGE_DEV_SUFFIX`.